MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -591,5 +594,29 @@ - (void)locationManager:(MMELocationManager *)locationManager didVisit:(CLVisit
     }
 }
 
+- (NSString *)locationAuthorizationStatus {
```

Comment:
this is a static function, it should be declared static, potentially in a category of CLLocationManager

Reply:
Added in these changes, thanks! Lemme know how it looks now. 🙇